src/foundry/client/data/documents/scene.d.mts

```diff
@@ -646,15 +641,14 @@ declare global {
 
         /**
          * The base (darkness level 0) ambience lighting data.
-         * @defaultValue See properties
          */
         base: fields.SchemaField<
           EnvironmentDataSchema<{ hue: 0; intensity: 0; luminosity: 0; saturation: 0; shadows: 0 }>
         >;
 
         /**
          * The dark (darkness level 1) ambience lighting data.
-         * @defaultValue See properties
+         * @privateRemarks The `hue` default is actually `257/360` but you can't do division in types. This precision should be more than sufficient.
```

Comment:
Okay but now when I write `Scene.Schema["dark"] extends SchemaField<EnvironmentDataSchema<{ hue: infer Hue }>> ? Hue : never` it'll be wrong :((((((((((